### ISSUE TITLE
can't compile under debian Jessie

### ISSUE THREAD

**Issue by highenergymagic** [devicenull/supermicro_ipmi_firmware#1]
when trying to compile under debian jessie, I get:

mkdir -p  /root/supermicro_ipmi_firmware/SDK/output/filesystem//bin /root/supermicro_ipmi_firmware/SDK/output/filesystem//sbin /root/supermicro_ipmi_firmware/SDK/output/filesystem//lib /root/supermicro_ipmi_firmware/SDK/output/filesystem//etc/init.d /root/supermicro_ipmi_firmware/SDK/output/filesystem//usr/local/etc/ntp

=================== kernel ===================

cd /root/supermicro_ipmi_firmware/SDK/Kernel/Host/HERMON/linux && LDFLAGS="" make
make[1]: Entering directory '/root/supermicro_ipmi_firmware/SDK/Kernel/Host/HERMON/linux'
Makefile:1310: *** mixed implicit and normal rules: deprecated syntax
  CHK     include/linux/version.h
/root/supermicro_ipmi_firmware/SDK/Kernel/Host/HERMON/linux/Makefile:410: *** mixed implicit and normal rules: deprecated syntax
/root/supermicro_ipmi_firmware/SDK/Kernel/Host/HERMON/linux/Makefile:1310: *** mixed implicit and normal rules: deprecated syntax
make[2]: *** No rule to make target 'silentoldconfig'.  Stop.
Makefile:457: recipe for target 'include/linux/autoconf.h' failed
make[1]: *** [include/linux/autoconf.h] Error 2
make[1]: Leaving directory '/root/supermicro_ipmi_firmware/SDK/Kernel/Host/HERMON/linux'
Makefile:45: recipe for target 'kernel' failed
make: *** [kernel] Error 2

I presume this is due to missing 2.6.17 kernel headers, but I can't find them

**Comment by devicenull** [comment]
It looks like your version of make might be too new.  I should note that this entire project was pulled from supermicro's GPL release, which was incomplete at best.

It's unlikely to be up to date, and I'm not sure how it's going to work these days.  It's definitely not something I'm really maintaining these days.

**Comment by highenergymagic** [comment]
you got a precompiled image kicing around? smash is evil.

**Comment by devicenull** [comment]
No, but this would probably be more useful for that anyway: https://github.com/devicenull/ipmi_firmware_tools

Just replace the binary with something more useful.
